<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTUploadPathWidget</class>
 <widget class="QWidget" name="HTUploadPathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HyperThought Upload Path</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="3">
    <spacer name="sourceSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>455</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="sourceLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Scope:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="sourceIdLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="pathLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="DeselectableTreeView" name="fileInfoTreeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="syncBtn">
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="sourceIdLE"/>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="sourceCombo">
     <item>
      <property name="text">
       <string>User</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Project</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="5" rowspan="5">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLineEdit" name="pathLE"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="propNameLabel">
     <property name="text">
      <string>Property</string>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>User: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="usernameLE"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeselectableTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">DeselectableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sourceCombo</tabstop>
  <tabstop>sourceIdLE</tabstop>
  <tabstop>pathLE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
